SUPPORT.md

```diff
@@ -0,0 +1,67 @@
+# The Lift Framework Support Policy
+
+Ongoing maintenance and development of the Lift Framework is a volunteer effort executed by a
+passionate group of Lift supporters known a the Lift Committers. With the assistance of
+outside contributors, the Lift Committers work to ensure that with each release of Lift we provide
+a selection of new features and bug fixes. The Lift Committers stand behind and provide ongoing
+support for the contributions they make to Lift, and a condition of accepting changes from outside
+contributors is that the Committers feel comfortable supporting those changes long term.
+
+Because maintenance of the Framework is a  volunteer effort and all of the Committers have busy
+lives with precious free time, they must be intentional with the community support they provide for
+Lift. To that end, the Committers have decided to embrace the following support policy for releases
+of Lift:
+
+* Lift ships a new release every six months, with three milestones and at least one Release
+Candidate building up to each release. Each normal release results in a bump of the middle number
+of the version (e.g. 3.1.0 to 3.2.0).
+* Brand new features are always a part of new releases. Breaking changes will only ship as a part
+of a major release, which results in a bump of the first number of the version (e.g. 2.6.0 to 3.0.0).
+* Bug fixes and minor performance improvements will be shipped for releases for 18 months after
```

Comment:
I would change this to “can, by consensus of the Committers, be shipped…” Feels like “will” is a bit strong here…